src/qt/bitcoingui.cpp

```diff
@@ -1452,23 +1452,72 @@ void BitcoinGUI::updateScraperIcon(int scraperEventtype, int status)
 
     int64_t nConvergenceTime = ConvergedScraperStatsCache.nTime;
 
-    std::string sExcludedProjects = {};
+    std::string sExcludedProjects;
+    std::string sIncludedScrapers;
+    std::string sExcludedScrapers;
+    std::string sScrapersNotPublishing;
 
-    bool bExcludedProjects = false;
+    bool bDisplayScrapers = false;
 
     // If the convergence cache has excluded projects...
-    if (ConvergedScraperStatsCache.vExcludedProjects.size())
+    if (!ConvergedScraperStatsCache.Convergence.vExcludedProjects.empty())
     {
-        bExcludedProjects = true;
-
-        for (const auto& iter : ConvergedScraperStatsCache.vExcludedProjects)
+        for (const auto& iter : ConvergedScraperStatsCache.Convergence.vExcludedProjects)
         {
             if (sExcludedProjects.empty())
                 sExcludedProjects += iter.first;
             else
                 sExcludedProjects += ", " + iter.first;
         }
     }
+    else
+    {
+        sExcludedProjects = "none";
```

Comment:
Wrap it in the `tr()` macro so it follows the current language.

Reply:
Isn't that actually less readable?

Reply:
I was thinking about building the entire string beforehand and passing just one argument to the tooltip instead, which I think is the opposite direction to what you are suggesting.

Reply:
You need to wrap it in the macro or else it the string scanner will not pick it up and include it for translation. So it will, in Swedish, become something like: "Exkluderade projekt: none".

Reply:
Ah.... you are right. We don't want to be English snobs.

Reply:
Done.

Reply:
Wait.... those were already going through the translation wrapper. they are an arg to setToolTip(tr(....).arg...)

Reply:
The args don't get run through `tr()`, do they? I think we need to `tr()` it first here, and then Qt will substitute the placeholder below with the localized string verbatim.

Reply:
I just looked closely at the macro and you are right I think. I will put the tr back in on the elses.